posts/squidpy-essentials/squidpy-essentials.qmd

```diff
@@ -0,0 +1,1084 @@
+---
+title: "Using Squidpy with AtoMx&#8482; SIP exports"
+author:
+  - name: Evelyn Metzger
+    orcid: 0000-0002-4074-9003
+    affiliations: 
+      - ref: nstg
+      - ref: eveilyeverafter
+toc: true
+toc-title: Contents
+toc-depth: 3
+toc-expand: 2
+toc-location: left
+number-sections: true
+number-depth: 3
+date: "2024-07-02"
+categories: [squidpy python, visualization, anndata]
+draft: false
+image: figures/fig-image1.png
+description: In this blog post, I’ll show how to prepare and analyze AtoMx&#8482; SIP-exported CosMx&#8482; SMI data with python\'s squidpy package.
+format: 
+  html:
+    theme: 
+      light: custom.scss
+      dark: darkly
+  pdf:
+    number-sections: true
+  docx:
+    toc: true
+    number-sections: true
+    highlight-style: github
+format-links: [pdf, docx]
```

Comment:
It builds when rendering via github actions automatically (you would need to control+shift+b to build locally if you would like the pdf to render; otherwise just the html is rendered). But in this post, I perhaps it makes sense _not_ to render pdf or word so I removed them.

55aca27bf4e9ada7f56d00d077b95f7fd0e20566